docs/Getting_Involved/Testing.rst

```diff
@@ -9,8 +9,15 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
+If you have not already, you can start by following
+the `coala Tutorial <Users/Tutorials/Tutorial.html>`_.
+
+Before you run tests, make sure you
+`install coala
+<Users/Install.html/>`_.
+
 To run tests, you first need to install some dependencies. This can be
-done by executing:
+done by executing in your coala directory:
 
 ::
 
```

Comment:
\+ for the commit message: we use normally a tag inside the shortlog (normally the filename modified). Also your shortlog should include an imperative tense verb describing the change, e.g.

`Testing.rst: Add directions about getting started`